src/calculator/routes.ts

```diff
@@ -19,6 +20,15 @@ calculatorRoutes.get('/twitter/user/:id', function(req, res, next) {
     })
 })
 
+calculatorRoutes.get('/scrapper/user-verified/:verified/user-joined/:joined', function(req, res, next) {
```

Comment:
The endpoint doesn't complies with the documented and expected one, read [this](https://world-white-web.atlassian.net/wiki/spaces/WWWA/pages/2752568/Twitter+user+credibility+without+Twitter+API).

Reply:
done